src/constants.ts

```diff
@@ -121,6 +121,26 @@ export const TEAM_BREADCRUMBS = [
   }
 ];
 
+export const TITLE_BREADCRUMBS = [
+  {
+    name: 'Головна',
+    link: ROUTES.HOME
+  },
+  {
+    name: 'Cеріли',
+    link: '#'
+  },
+
+  {
+    name: 'Субтитровані серіали',
+    link: '#'
+  },
+  {
+    name: 'Константин: Місто Демонів',
```

Comment:
The last item in TITLE_BREADCRUMBS should be added dynamically(from API as title name), so remove it from here and paste it using spread operator directly in the component